web/src/features/hfiCalculator/components/FireStartsDropdown.tsx

```diff
@@ -0,0 +1,44 @@
+import { TextField } from '@material-ui/core'
+import { Autocomplete } from '@material-ui/lab'
+import {
+  FireStarts,
+  FIRE_STARTS_SET,
+  lowestFireStarts
+} from 'features/hfiCalculator/slices/hfiCalculatorSlice'
+import { isEqual, isNull } from 'lodash'
+import React from 'react'
+
+export interface FireStartsDropdownProps {
+  fireStarts: FireStarts | undefined
+  areaName: string
+  dayOffset: number
+  setFireStarts: (areaName: string, dayOffSet: number, newFireStarts: FireStarts) => void
```

Comment:
```suggestion
  setFireStarts: (areaName: string, dayOffset: number, newFireStarts: FireStarts) => void
```

Reply:
very picky & not really important - just suggesting it for consistency with how "Offset" is used elsewhere in the file

Reply:
Done